03-interactive-site/02-ux-design/02-ux-design/README.md

```diff
@@ -46,3 +46,16 @@ metas de los usuarios. Pero, sin dejar de lado las metas de los negocios.
 Para entender un poco más sobre el proceso de UX Design mira este video:
 
 [![](https://lh5.googleusercontent.com/OHW33sLkmWQv1eljJlycGHozE-ozx6WXVE-rnYKOmke4hqXzXyKhD67dGEDw_ILwsyFMiYT29n4ECRatw5Gfd4tnjs8Q3HZh5nT8qZOiffp7HuLZSDA_IMzD5MoGqvI_2f71IfGOgG4)](https://youtu.be/SBIvPTSy1QM)
+
+## ¿Por qué estamos aprendiendo UX?
+Usualmente vamos a querer que los usuarios de nuestros productos (digitales o físicos) estén felices cuando consumen lo que ofrecemos.
+
+¿Sentarse en una silla ultra comoda o un banco de madera?
+¿Comprar en un starbucks o en una cafetería local?
+¿Usar Facebook o Myspace?
+
+La experiencia al usar nuestros productos es directamente proporcional a su consumo.
+
+Al tener una mejor UX tenemos mas consumo.
+
+Y viceversa.
```

Comment:
Me gusta el texto y las preguntas. Dos temas que acotar, quizás en lugar de consumo pondría satisfacción. Y esta satisfacción se podría medir en mayor consumo en una 'for profit' o en mayor impacto en una 'non for profit'. La parte final sí es tricky porque un mayor consumo no necesariamente significa un mejor UX. Aquí un ejemplo:
>Page Views kinda’, sorta’ measures consumption. But are a lot of Page Views per visit a good thing (“the visitor loved our site so much!”), a bad thing (“our site is so bad it takes 23 pages to find what you’re looking for!”), or a horrible thing (“After a 23-page hunt, the visitor gave up!”)? 
(https://www.thinkwithgoogle.com/marketing-resources/data-measurement/you-are-what-you-measure/)

Reply:
@lalogf @indie-rok voy a aceptar el PR para que se vayan corrigiendo otros temas de redacción, pero les agradecería se pongan de acuerdo en cómo mejorar esta parte y quien de los dos manda un PR con las mejoras 😃 

Muchas gracias de antemano por darse el tiempo de revisar este contenido! 💪